control/control_origination_test.go

```diff
@@ -0,0 +1,60 @@
+package control
+
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/opencontrol/fedramp-templater/fixtures"
+)
+
+var _ = Describe("controlOrignation", func() {
+	Describe("newControlOrignation", func() {
+		It("should return", func() {
+			doc := fixtures.LoadSSP("FedRAMP_ac-2-1_v2.1.docx")
+			defer doc.Close()
+			tables, err := doc.SummaryTables()
+			Expect(err).NotTo(HaveOccurred())
+			st := NewSummaryTable(tables[0])
+			co, err := newControlOrigination(st)
+			// Check error
+			Expect(err).ToNot(HaveOccurred())
+			// Check number of control origination.
+			Expect(len(co.origins)).To(Equal(7))
+
+			// Find the unchecked service provided corporate origination.
+			Expect(co.origins[serviceProviderCorporateOrigination].GetTextValue()).
+				To(ContainSubstring(serviceProviderCorporateOrigination))
+			Expect(co.origins[serviceProviderCorporateOrigination].IsChecked()).To(Equal(false))
```

Comment:
Not sure if these checks are providing a lot of value...feels like we're getting at it better in https://github.com/opencontrol/fedramp-templater/blob/5194ce95614ee8b0fc94af8e994ee5a4a225f33c/control/summary_table_test.go#L70-L81, so I'd just remove.